lang/python/python3-zipp/patches/001-fix-scm-version.patch

```diff
@@ -0,0 +1,13 @@
+Index: zipp-0.5.1/setup.py
+===================================================================
+--- zipp-0.5.1.orig/setup.py
++++ zipp-0.5.1/setup.py
```

Comment:
Would you please remove it the exact version? If there is a new release, you will need to rebase the patch. There's a chance that you will don't need to do it for each version. In that case, the refresh should be enough.

Reply:
Removed